.claude/hooks/poll-pr-status.sh

```
@@ -0,0 +1,321 @@
+    reviews="$(gh api "repos/${repo_full}/pulls/${PR_NUM}/reviews" \
+      --jq '[.[] | select(.user.login == "coderabbitai[bot]")] | sort_by(.submitted_at) | last' \
+      2>/dev/null || echo "")"
+
+    if [[ -z "$reviews" || "$reviews" == "null" ]]; then
+      sleep "$CODERABBIT_POLL_INTERVAL"
+      elapsed=$((elapsed + CODERABBIT_POLL_INTERVAL))
+      continue
+    fi
+
+    # Extract review ID and state
+    local review_id review_state review_body
```

Shellcheck (0.11.0)

[warning] review_body appears unused. Verify use (or export if used externally).

(SC2034)